src/reactviews/pages/UserSurvey/userSurveyPage.tsx

```diff
@@ -0,0 +1,303 @@
+                                display: "flex",
+                                flexDirection: "column",
+                            }}
+                        >
+                            {"0"}
+                            <br />
+                            <Text
+                                style={{
+                                    position: "absolute",
+                                    width: "100px",
+                                    top: "30px",
+                                    left: "0px",
+                                    fontSize: "10px",
+                                }}
+                                size={200}
+                            >
+                                {locConstants.userFeedback.notLikelyAtAll}
+                            </Text>
+                        </div>
+                    }
+                />
+                <Radio value={"1"} label={"1"} />
+                <Radio value={"2"} label={"2"} />
+                <Radio value={"3"} label={"3"} />
+                <Radio value={"4"} label={"4"} />
+                <Radio value={"5"} label={"5"} />
+                <Radio value={"6"} label={"6"} />
+                <Radio value={"7"} label={"7"} />
+                <Radio value={"8"} label={"8"} />
+                <Radio value={"9"} label={"9"} />
+                <Radio
+                    value={"10"}
+                    label={
+                        <div
+                            style={{
+                                position: "relative",
+                                display: "flex",
+                                flexDirection: "column",
+                            }}
+                        >
+                            {"10"}
+                            <br />
+                            <Text
+                                style={{
+                                    position: "absolute",
+                                    width: "max-content",
+                                    top: "30px",
+                                    right: "0px",
+                                    fontSize: "10px",
+                                }}
+                                size={200}
+                            >
+                                {locConstants.userFeedback.extremelyLikely}
+                            </Text>
+                        </div>
+                    }
+                />
+            </RadioGroup>
+        </Field>
+    );
+};
+
+export const TextAreaQuestion = ({ question, onChange }: QuestionProps) => {
+    const userSurveryProvider = useContext(UserSurveyContext);
+    if (!userSurveryProvider) {
+        return undefined;
+    }
+    return (
+        <Field
+            required={question.required ?? false}
+            label={<Text weight="bold">{question.label}</Text>}
+        >
+            <Textarea
+                placeholder={
+                    question.placeholder ??
```

Comment:
We should _always_ display the privacy disclaimer.  Perhaps that goes above the `<Textarea>` and we maintain the optional placeholder text here.